proxy/src/main/java/com/velocitypowered/proxy/protocol/packet/brigadier/EnumArgumentProperty.java

```diff
@@ -0,0 +1,23 @@
+package com.velocitypowered.proxy.protocol.packet.brigadier;
```

Comment:
Nit: I'd move this to `com.velocitypowered.proxy.protocol.packet.brigadier.forge`

Reply:
The license header is missing here too.